packages/components/header/template.njk

```diff
@@ -32,138 +27,60 @@
         {%- endif %}
       </a>
       {%- else -%}
-      <a class="nhsuk-header__link{% if params.service %} nhsuk-header__link--service {% endif %}" href="{% if params.homeHref %}{{ params.homeHref }}{% else %}/{% endif %}" aria-label="{% if params.ariaLabel %}{{ params.ariaLabel }}{% else %}NHS homepage{% endif %}">
+      <a class="nhsuk-header__link" href="{% if params.homeHref %}{{ params.homeHref }}{% else %}/{% endif %}" aria-label="{% if params.ariaLabel %}{{ params.ariaLabel }}{% else %}NHS homepage{% endif %}">
```

Comment:
Rather than defaulting to `/`,  I wonder if this should not link at all if no `params.homeHref` is present? Or alternatively it could link to `https://www.nhs.uk` but perhaps having no link at all would be safer, to avoid it being overlooked?

Or I guess a third option would be to display a visible error message to force a link to be specified?